aodntools/timeseries_products/hourly_timeseries.py

```diff
@@ -521,21 +524,17 @@ def hourly_aggregator(files_to_aggregate, site_code, qcflags, file_path ='./'):
         for stat_method in function_stats:
             variable_stat_name = variable + "_" + stat_method
             ancillary_variables_attr += [variable_stat_name]
-            try:
-                nc_aggregated[variable_stat_name].attrs['standard_name'] = nc_aggregated[variable].attrs['standard_name']
-            except:
-                pass
-            nc_aggregated[variable_stat_name].attrs['long_name'] = stat_method + ' data value in the bin, after rejection of flagged data'
+
             if stat_method == 'count':
                 nc_aggregated[variable_stat_name].attrs['units'] = 1
-                try:
-                    nc_aggregated[variable_stat_name].attrs['standard_name'] = nc_aggregated[variable].attrs[
-                        'standard_name'] + '_number_of_observations'
-                except:
-                    pass
-
             else:
                 nc_aggregated[variable_stat_name].attrs['units'] = variable_attributes[variable]['units']
+
+            if 'standard_name' in nc_aggregated[variable].attrs:
+                nc_aggregated[variable_stat_name].attrs['standard_name'] = nc_aggregated[variable].attrs['standard_name']
+                nc_aggregated[variable+'_count'].attrs['standard_name'] = nc_aggregated[variable].attrs['standard_name'] + '_number_of_observations'
```

Comment:
That last bit should be  + ' number_of_observations' (first character is space, not '_')

Reply:
ok